generative/maisi/README.md

```diff
@@ -0,0 +1,84 @@
+# Medical AI for Synthetic Imaging (MAISI)
+This example demonstrates the applications of training and validating NVIDIA MAISI (Medical AI for Synthetic Imaging), a 3D Latent Diffusion Model (LDM) capable of generating large CT images accompanied by corresponding segmentation masks. It supports variable volume size and voxel spacing and allows for the precise control of organ/tumor size.
```

Comment:
We have defined MAISI in the title. We can probably skip `MAISI (Medical AI for Synthetic Imaging)`

Reply:
I have updated the part. Please proceed. Thank you.